models/articles.model.js

```diff
@@ -37,9 +37,39 @@ exports.selectArticles = (sort_by = "created_at", order = "DESC") => {
   );
 
   return db.query(queryString).then(({ rows }) => {
-    // if (rows.length === 0) {
-    //   return Promise.reject({ status: 400, msg: "does not exist" });
-    // }
+ 
     return rows;
   });
 };
+
+exports.selectComments = (sort_by = "created_at", order = "DESC", article_id) => {
+  // Greenlist
+  const validOrders = ["ASC", "DESC"];
```

Comment:
You've skipped ahead a bit here. I would ideally remove or comment out the sortby and orderby for now.